scalability_and_performance/ztp-far-edge/ztp-upgrading-gitops.adoc

```diff
@@ -0,0 +1,30 @@
+:_content-type: ASSEMBLY
+[id="ztp-upgrading-gitops"]
+= Upgrading GitOps ZTP
+include::_attributes/common-attributes.adoc[]
+:context: ztp-upgrading-gitops
+
+toc::[]
+
+You can upgrade the Gitops zero touch provisioning (ZTP) infrastructure independently from the underlying cluster, {rh-rhacm-first}, and {product-title} version running on the managed clusters.
```

Comment:
You could minimise the sentence by changing "{product-title} version running on the managed clusters" to the following:

> the managed {product-title} clusters